src/witan/app/handler.clj

```diff
@@ -101,9 +112,11 @@
                            :middlewares [cors-mw]
                            :summary "sign up"
                            (signup user))
-            (sweet/GET* "/user/:id" []
-                         :summary "Get user by ID"
-                         (not-implemented))
+            (sweet/GET* "/me" []
+                        :middlewares [cors-mw token-auth-mw pass-identity]
+                        :query-params [identity :- java.util.UUID]
```

Comment:
fixed it, had to look through code to find a way to retrieve the request (but was fairly simple in the end)